prx_utilities/rigid_body_info_msg[] rigid_body_array
---
